metadata.patch

```diff
@@ -0,0 +1,63 @@
+--- a/butt/usr/share/metainfo/de.danielnoethen.butt.metainfo.xml
++++ b/butt/usr/share/metainfo/de.danielnoethen.butt.metainfo.xml
+@@ -1,23 +1,53 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop-application">
+   <id>de.danielnoethen.butt</id>
+-  
++
+   <name>butt</name>
+   <summary>Icecast and Shoutcast streaming client</summary>
+-  
++  <developer_name>Daniel Nöthen</developer_name>
++  <developer id="de.danielnoethen">
++    <name>Daniel Nöthen</name>
++  </developer>
++
+   <metadata_license>MIT</metadata_license>
+   <project_license>GPL-2.0-or-later</project_license>
+-  
++
+   <description>
++    <p>
++      NOTE: This wrapper is not verified by, affiliated with, or supported by the developer. Therefore, and so to offer the best support and experience for the Flatpak wrapper please report ALL issues at the Flathub issue tracker (linked by the issue button/link on this page). This allows with issues to be dealt with correctly. For more info press the issues button/link on this page. 
++    </p>
+     <p>
+       butt (broadcast using this tool) is an easy to use, multi OS streaming tool. It supports SHOUTcast and Icecast and runs on Linux, Mac OS X and Windows (64 Bit only since version 0.1.20). The main purpose of butt is to stream live audio data from your computers Mic or Line input to a Shoutcast or Icecast server. If you want you can also record your session.
+     </p>
+   </description>
+-  
+-  <launchable type="desktop-id">butt.desktop</launchable>
++
++  <releases>
++    <release version="1.44.0" date="2024-12-01">
++      <description>
++        <ul>
++          <li>BUTT can now request the current song from an URL (website)</li>
++          <li>It is now possible to instruct BUTT to overwrite existing recordings</li>
++          <li>Add a new display mode which shows "On Air" and if available the current listeners</li>
++          <li>Fix memory corruption after canceling a connection attempt when using a different application language than English</li>
++          <li>Improve compatibility of UTF-8 in song names with some servers by making the update request compliant to RFC3986</li>
++          <li>Increase font size of the "Listeners" label to improve readability from far distances</li>
++          <li>Change the default answer from "Yes" to "Cancel" when asked to overwrite existing recordings to prevent accidental overwriting of existing recordings</li>
++        </ul>
++      </description>
++    </release>
++  </releases>
++
++  <launchable type="desktop-id">de.danielnoethen.butt.desktop</launchable>
+   <screenshots>
+     <screenshot type="default">
+       <image>https://danielnoethen.de/butt/butt_ubuntu.png</image>
++      <caption>Screenshot of the main application UI</caption>
+     </screenshot>
+   </screenshots>
++
++  <url type="homepage">https://danielnoethen.de/butt/</url>
++  <url type="bugtracker">https://github.com/flathub/de.danielnoethen.butt/issues/</url>
```

Comment:
this URL doesn't exist. If there is no upstream bug tracker, remove this.

Reply:
Bug tracking is officially done via email so i will remove this linl